modules/client/examples.md

```diff
@@ -2444,3 +2551,576 @@ console.log(members);
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+
+const client = new MultisigClient(contextPlugin);
+
+console.log(client);
+```
+
+### Creating a multisig proposal
+
+```ts
+import {
+  MultisigClient,
+  Context,
+  ContextPlugin,
+  ProposalCreationSteps,
+  ProposalMetadata,
+  Client
+} from "@aragon/sdk-client";
+import { CreateMultisigProposalParams, IWithdrawParams } from "../../src";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create a multisig client
+const client:Client = new Client(context);
+// Create a multisig client
+const multisigClient:MultisigClient = new MultisigClient(contextPlugin);
+
+const metadata: ProposalMetadata = {
+  title: "Test Proposal",
+  summary: "This is a short description",
+  description: "This is a long description",
+  resources: [
+    {
+      name: "Discord",
+      url: "https://discord.com/...",
+    },
+    {
+      name: "Website",
+      url: "https://website...",
+    },
+  ],
+  media: {
+    logo: "https://...",
+    header: "https://...",
+  },
+};
+
+const ipfsUri = await multisigClient.methods.pinMetadata(metadata);
+const withdrawParams: IWithdrawParams = {
+  recipientAddress: "0x1234567890123456789012345678901234567890",
+  amount: BigInt(10),
+  tokenAddress: "0x1234567890123456789012345678901234567890",
+  reference: "test",
+};
+const daoAddress = "0x1234567890123456789012345678901234567890";
+
+const withdrawAction = await client.encoding.withdrawAction(
+  daoAddress,
+  withdrawParams,
+);
+
+const proposalParams: CreateMultisigProposalParams = {
+  pluginAddress: "0x1234567890123456789012345678901234567890",
+  metadataUri: ipfsUri,
+  actions: [withdrawAction],
+};
+
+const steps = multisigClient.methods.createProposal(proposalParams);
+for await (const step of steps) {
+  try {
+    switch (step.key) {
+      case ProposalCreationSteps.CREATING:
+        console.log(step.txHash);
+        break;
+      case ProposalCreationSteps.DONE:
+        console.log(step.proposalId);
+        break;
+    }
+  } catch (err) {
+    console.error(err);
+  }
+}
+```
+
+### Approve a multisig proposal
+
+```ts
+import {
+  ApproveMultisigProposalParams,
+  ApproveProposalStep,
+  Context,
+  ContextPlugin,
+  MultisigClient,
+} from "@aragon/sdk-client";
+import { contextParams } from "../00-client/00-context";
+
+// Create a simple context
+const context: Context = new Context(contextParams);
+// Create a plugin context from the simple context
+const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
+// Create an multisig client
+const client = new MultisigClient(contextPlugin);
+
+const approveParams: ApproveMultisigProposalParams = {
+  proposalId:
+    "0x1234567890123456789012345678901234567890000000000000000000000001",
```

Comment:
Warning: the pluginId format is not merged at all. Actually the discussion was closed, then reopened